src/Components/WebAssembly/WebAssembly/src/targets/Microsoft.AspNetCore.Components.WebAssembly.props

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <BlazorWebAssemblyJSPath>$(MSBuildThisFileDirectory)blazor.webassembly.js</BlazorWebAssemblyJSPath>
     <BlazorRoutingEnableRegexConstraint Condition="'$(BlazorRoutingEnableRegexConstraint)' == ''">false</BlazorRoutingEnableRegexConstraint>
+    <WebAssemblyHotReloadCapabilities>Baseline;AddMethodToExistingType;AddStaticFieldToExistingType;NewTypeDefinition;ChangeCustomAttributes;AddInstanceFieldToExistingType;GenericAddMethodToExistingType;GenericUpdateMethod;UpdateParameters;GenericAddFieldToExistingType</WebAssemblyHotReloadCapabilities>
```

Comment:
Should this go into the `\build\*.props` file for the Hot Reload agent long term?

/cc: @maraf

Reply:
I don't think so. This is specific to WASM. In other cases the agent retrieves the information from the runtime. The problem with WASM is that the runtime (and the agent) might not be loaded yet into the browser at the point we need the information.